cvelib/cli.py

```diff
@@ -298,32 +301,54 @@ def publish(ctx: click.Context, cve_id: str, cve_json_str: str, print_raw: bool)
 def reject(ctx: click.Context, cve_id: str, cve_json_str: str, print_raw: bool) -> None:
     """Reject a CVE record for a reserved or published CVE ID.
 
-    Will update if the CVE record already exists.
+    If the CVE is already rejected, this action will update its record if one is supplied.
+    A rejected CVE with a record can only be moved to the published state (see `cve publish`). A
+    rejected CVE without a record can be moved to the reserved state.
+
+    Example:
 
     \b
-    cve reject 'CVE-2022-1234' --json '{"rejectedReasons": [{"lang": "en", "value": "A reason."}]}'
+    cve reject CVE-2022-1234 -j '{"rejectedReasons": [{"lang": "en", "value": "A reason."}]}'
```

Comment:
I see you remove the quotes around the first arg value here - let's keep these to distinguish argument names from values, or else remove them from README.md as well.

Reply:
It should be clear that it's an argument, I'll remove it in the readme as well.